src/callbacks/liquidity/BaselineV2/BaselineAxisLaunch.sol

```diff
@@ -286,10 +313,14 @@ contract BaselineAxisLaunch is BaseCallback, Policy, Owned {
                 address(RESERVE), address(bAsset), auctionPrice, 10 ** baseTokenDecimals
             );
             activeTick = TickMath.getTickAtSqrtRatio(sqrtPriceX96);
-        }
 
-        // Initialize the Baseline pool at the tick determined by the auction price
-        BPOOL.initializePool(activeTick);
+            // Check that the pool is initialized at the active tick
+            // This is to ensure that the pool is initialized at the auction price
+            (, int24 poolCurrentTick,,,,,) = BPOOL.pool().slot0();
+            if (poolCurrentTick != activeTick) {
```

Comment:
I believe the activeTick accounts for tick rounding, but just want to make sure. The comment should be more like "initialized at the tick closest to the auction price".

Reply:
`TickMath.getTickAtSqrtRatio()` (which sets `activeTick`) does not handle rounding. The tick returned by `slot0()` also is not rounded.

Have modified the comment.